lib/urlwatch/handler.py

```diff
@@ -41,6 +41,8 @@
 
 
 class JobState(object):
+    resouces = {}
```

Comment:
Typo: `resources`

Reply:
Also, are we sure it should be a class member (one per class, not one per instance)?

Reply:
In the design I have in mind, yes, it should be a class member, i.e. it's a globally unique static variable.
The point of it is to share expensive resources among different jobs. In browser job's case, these are the browser instance and async event loop.